### PR TITLE
[AGENT-4323] Add jenkins job test and build script

### PR DESCRIPTION
**Pull request by oleksandr-saienko**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Added Jenkins automation for Test and Build process
Also publishing to TestPyPi

## Rationale
To speedup development process Jenkins Test and Build Job added

### CODE REVIEW

**Review by dulcardo — APPROVED** [review]

**Comment by oleksandr-saienko** [comment]
Jenkins run publish to pypi

**Comment by oleksandr-saienko** [comment]
Jenkins run publish to pypi

File: setup.py [hunk 1]
```diff
@@ -27,10 +27,10 @@
         'datarobot_provider.sensors',
         'datarobot_provider.operators',
     ],
-    install_requires=['apache-airflow>=2.0', 'datarobot>=2.28.0'],
+    install_requires=['apache-airflow>=2.3.4', 'datarobot>=2.28.0'],
     setup_requires=['setuptools', 'wheel'],
-    author='Andrius Senulis',
-    author_email='andrius.senulis@datarobot.com',
+    author='Oleksandr',
+    author_email='oleksandr.saienko@datarobot.com',
```
Comment (elatt) [inline setup.py:33]:
```suggestion
    author='DataRobot',
    author_email='support@datarobot.com',
```

**Comment by oleksandr-saienko** [comment]
Jenkins run publish to pypi

**Comment by oleksandr-saienko** [comment]
Jenkins run publish to pypi

File: jenkins/bash_scripts/add_build_version.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (elatt) [inline jenkins/bash_scripts/add_build_version.sh:1]: This isn't the right approach; you can delete this file in the next iteration of the publishing.

**Review by elatt — APPROVED** [review]
LGTM

**Pull request merged**